lib/matplotlib/axes/_axes.py

```diff
@@ -4522,13 +4522,35 @@ def arrow(self, x, y, dx, dy, **kwargs):
         """
         Add an arrow to the axes.
 
-        Call signature::
+        Draws arrow on specified axis from (`x`, `y`) to (`x` + `dx`,
+        `y` + `dy`). Uses FancyArrow patch to construct the arrow.
+
+        Parameters
+        ----------
+        x : float
+            X-coordinate of the arrow base
+        y : float
+            Y-coordinate of the arrow base
+        dx : float
+            Length of arrow in x-coordinate
+        dy : float
+            Length of arrow in y-coordinate
+
+        Return
```

Comment:
There is a small mistake in the title of this section. It should be Returns (with an s)